src/components/input/code/templates/inputTemplate.jsx

```diff
@@ -0,0 +1,46 @@
+var React = require('react');
+
+module.exports = function (props){
+
+	var classes = classNames('input-group', {
```

Comment:
You're using a function called `classNames` here but it's not been defined or required in.